CHANGELOG.md

```diff
@@ -43,6 +44,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Add SMA view with SMN Copy Number data
 - Easier to assign OMIM diagnoses from case page
 - OMIM terms and specific OMIM term page
+- Option to use alternative flask config when running `scout serve`
```

Comment:
Could it be that you didn't remember that you already modified the changelog in this PR?